transforms/csdl/preprocess_csdl.xsl

```diff
@@ -5,6 +5,7 @@
     <xsl:output method="xml" indent="yes"/>
     <xsl:strip-space elements="*"/> <!-- Remove empty space after deletions. -->
     <xsl:param name="remove-capability-annotations">True</xsl:param>
+    <xsl:param name="remove-innererror-description">True</xsl:param>
```

Comment:
I suggest that we name this `add*` since that is what we are doing here. The default should be `False` since we want existing use of this transform to continue without change.

```suggestion
    <xsl:param name="add-innererror-description">False</xsl:param>
```